### DISCUSSION TITLE
Installation auch an TSUN M800 ohne Panels möglich?

### DISCUSSION THREAD

**Discussion by Tibercad** [tbnobody/OpenDTU#1028] [General]
Hallo,
ich habe es noch nicht geschafft meine gelieferten Paneele auf dem Dach zu montieren. , jedoch habe ich das Modul bereits aus meiner Bastelkiste bauen können.
Nun würde ich das auch gleich einrichten. Da ich die Paneele auch nicht alleine herumwuchten möchte, muss ich warten bis ein freund Zeit hat mir zu helfen.

Es stellt sich die Frage für mich, kann ich den Wechselrichter ohne angeschlossene Paneele einstecken um dann das OpenDTU zu konfigurieren oder geht der Wechselrichter wirklich erst ganz an wenn die Paneele angeschlossen sind?

Grüße

**Comment by teletapi** [comment]
ohne panele startet der WR nicht

**Comment by Tibercad** [comment]
Ok, würde er starten wenn ich die Paneele anschließe diese aber noch keine Spannung liefern? Ich habe die Kartons derzeit im Dachspitz liegen, also Anschließen wäre möglich aber Sonneneinstrahlung ist noch nicht möglich.

**Reply by Solarteur** [reply]
Die HM-Wechselrichter arbeiten nur, wenn die angeschlossenen Module eine gewisse Mindestspannung liefern (s. Datenblatt).

**Reply by DPO99** [reply]
Müsstest ein Panel raus stellen, das sollte reichen aber im Karton wird das nix.

**Reply by Tibercad** [reply]
Mist, da oben hab ich nur ein winziges Fenster. Muss ich mal in der Mittagszeit versuchen, damit zumindest ein Bereich kurzzeitig in der Sonne liegt. Glaub da ist es besser ich warte bis die Dinger auf dem Dach sind.
